testing/ef-tests/src/cases/blockchain_test.rs

```diff
@@ -319,8 +319,13 @@ fn run_case(case: &BlockchainTest) -> Result<(), Error> {
 
     // Now validate using the stateless client if everything else passes
     for (block, execution_witness) in program_inputs {
-        stateless_validation(block, execution_witness, chain_spec.clone())
-            .expect("stateless validation failed");
+        stateless_validation(
+            block,
+            execution_witness,
+            chain_spec.clone(),
+            EthEvmConfig::new(chain_spec.clone()),
```

Comment:
This doesn't look correct -- We _could_ avoid this is ConfigureEvm exposed a `chain_spec` function but I'm still thinking about whether that breaks any abstractions